core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2127,6 +2127,16 @@ private void renameInternal(RpcContext rpcContext, LockedInodePath srcInodePath,
 
     // Now we remove srcInode from its parent and insert it into dstPath's parent
     renameInternal(rpcContext, srcInodePath, dstInodePath, false, context);
+
+    LOG.info("Checking if persist after rename is enabled");
```

Comment:
INFO seems high. Did you mean DEBUG? Same for the rest of the statements below

Reply:
Whoops the commit that removed these seemed to have disappeared